cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/class-plugin.php

```diff
@@ -266,18 +266,19 @@ private function is_notice_component( $component ) {
 	 */
 	public function setup() {
 		$this->set_config();
-		$components = array_filter( $this->components, array( $this, 'is_setup_component' ) );
-		array_map(
-			function ( $component ) {
-				/**
-				 * Component that implements Component\Setup.
-				 *
-				 * @var  Component\Setup $component
-				 */
-				$component->setup();
-			},
-			$components
-		);
+
+		/**
+		 * Component that implements Component\Setup.
+		 *
+		 * @var  Component\Setup $component
+		 */ 
+		foreach ( $this->components as $key => $component ) {
```

Comment:
This DocBlock was here before this PR.

But this might be a good opportunity to remove it. Since this isn't a function or property, I'm not sure this needs a DocBlock.

Reply:
It helps with editor navigation, so it might be useful to leave it here

Reply:
OK, in that case, could it be about `$component->setup();` like before?

```php
/**
 * Component that implements Component\Setup.
 *
 * @var Component\Setup $component
 */
$component->setup();